.github/workflows/newman.yml

```diff
@@ -0,0 +1,78 @@
+name: Postman Tests
+
+on:
+  push:

```

Comment:
trigger on push only when collections are updated

Reply:
should be triggered also when backend is updated